registry/quickswap/calldata-QuickSwap.json

```diff
@@ -0,0 +1,555 @@
+                "$id": "swapExactETHForTokensSupportingFeeOnTransferTokens",
+                "intent": "Swap",
+                "fields": [
+                    {
+                        "path": "params.amountOutMin",
+                        "label": "Minimum amount to Receive",
+                        "format": "tokenAmount",
+                        "params": {
+                            "tokenPath": "params.path[1]"
+                        }
+                    },
+                    {
+                        "path": "params.to",
+                        "label": "Beneficiary",
+                        "format": "addressName"
+                    },
+                    {
+                        "path": "params.deadline",
+                        "label": "Deadline",
+                        "format": "date",
+                        "params": {
+                            "encoding": "blockheight"
+                        }
+                    }
+                ],
+                "required": ["params.amountOutMin", "params.to"]
+            },
+            "0xe8e33700": {
+                "$id": "addLiquidity",
+                "intent": "Add Liquidity",
+                "fields": [
+                    {
+                        "path": "params.amountADesired",
+                        "label": "Desired amount",
+                        "format": "tokenAmount",
+                        "params": {
+                            "tokenPath": "params.tokenA"
+                        }
+                    },
+                    {
+                        "path": "params.amountAMin",
+                        "label": "Minimum amount",
+                        "format": "tokenAmount",
+                        "params": {
+                            "tokenPath": "params.tokenA"
+                        }
+                    },
+                    {
+                        "path": "params.amountBDesired",
+                        "label": "Desired amount",
+                        "format": "tokenAmount",
+                        "params": {
+                            "tokenPath": "params.tokenB"
+                        }
+                    },
+                    {
+                        "path": "params.amountBMin",
+                        "label": "Minimum amount",
+                        "format": "tokenAmount",
+                        "params": {
+                            "tokenPath": "params.tokenB"
+                        }
+                    },
+                    {
+                        "path": "params.to",
+                        "label": "Beneficiary",
+                        "format": "addressName"
+                    },
+                    {
+                        "path": "params.deadline",
+                        "label": "Deadline",
+                        "format": "date",
+                        "params": {
+                            "encoding": "blockheight"
```

Comment:
Quickswap looks like a duplicate of Uniswap Router02, and in Router02 addLiquidity looks like its encoded using a uni timestamp (https://docs.uniswap.org/contracts/v2/reference/smart-contracts/router-02#addliquidity). The encoding here should probably be "timestamp"